content/millennium/r4/conformance/structure-definition.md

```diff
@@ -0,0 +1,86 @@
+[`financial-transaction-account-number`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-account-number?_format=json)   |  A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+[`financial-transaction-alias`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-alias?_format=json)                     |  Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+[`financial-transaction-allocation`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation?_format=json)           |  Defines how the payment or adjustment is to be allocated across other resources.
+[`financial-transaction-amount`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount?_format=json)                   |  The total amount of the financial transaction.
+[`financial-transaction-card-brand`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-card-brand?_format=json)           |  Identifies the brand of credit card when credit card is used as a payment method.
+[`financial-transaction-date`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-date?_format=json)                       |  Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+[`financial-transaction-location`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location?_format=json)               |  Client configured value representing the location or workflow that the payment was received in.
+[`financial-transaction-method`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method?_format=json)                   |  Describes the method of payment for the financial transaction.
+[`financial-transaction-tendered-amount`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount?_format=json) |  The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+[`financial-transaction-type`](https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type?_format=json)                       |  The classification of the transaction.
+[`account-related-parts`](https://fhir-ehr.cerner.com/r4/StructureDefinition/account-related-parts?_format=json)                                 |  A reference to other related Accounts.
+[`account-state`](https://fhir-ehr.cerner.com/r4/StructureDefinition/account-state?_format=json)                                                 |  The status of the Account within the billing or correspondence workflow.
```

Comment:
move links to bottom of file

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-56528e5fbe9efbbf400be5f023a9a599L72)